src/popup/views/app.vue

```diff
@@ -180,7 +180,7 @@
            tabindex="0"
            @click="onRemoveScript"/>
       <div v-if="!activeExtras.data.config.removed && canUpdate(activeExtras.data)"
-           v-text="i18n('titleScriptUpdated')"
+           v-text="i18n('buttonUpdate')"
```

Comment:
Should this be `buttonCheckForUpdates`?

Reply:
Nope, because is this button used as direct action.

![r235234t](https://user-images.githubusercontent.com/2406819/194751809-5ffc7336-bdec-4067-bf18-57448701d39a.png)

So technically, all languages except English should add new translations for this action.

Given that the former `buttonUpdate` is now renamed as `buttonCheckForUpdates`, we should be able to submit new translations for this new `buttonUpdate` via Transifex, right? 🤔

Reply:
@gera2ld I mean the button always showed "Update" since BETA v2.13.1.2, not "Check for Updates", so I wanted to let it as is, but to be able to add a new translation for it.

Reply:
Ah I see what you mean. Could you run `yarn i18n` to normalize the translation files?

Reply:
Done. 👍